docs/types/mapping-types.rst

```diff
@@ -64,6 +67,25 @@ contract that returns the value at the specified address.
 The example below is a simplified version of an
 `ERC20 token <https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol>`_.
 ``_allowances`` is an example of a mapping type inside another mapping type.
+
+In the example below, the optional ``KeyName`` and ``ValueName`` are provided for the mapping. 
+It does not make affect any contract functionality or bytecode, it only sets the ``name`` field
+for the inputs and outputs in the ABI for the mapping's getter.
+
+.. code-block:: solidity
+
+    // SPDX-License-Identifier: GPL-3.0
+    pragma solidity >=0.8.17;
```

Comment:
```suggestion
    pragma solidity ^0.8.17;
```
We like to restrict the pragmas in the documentation. Mainly because the documentation will still be available and there is no guarantee that the code with `>= 0.8.17` will compile on `0.9.0` for example.